### PR TITLE
libtommath: install pkgconfig file

### PR DESCRIPTION
**Pull request by chrstphrchvz**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.6.5
Xcode 14.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: math/libtommath/Portfile [hunk 1]
```diff
@@ -52,6 +52,12 @@ destroot {
         ${destroot}${prefix}/lib
     ln -s libtommath.${version}.dylib ${destroot}${prefix}/lib/libtommath.0.dylib
     ln -s libtommath.${version}.dylib ${destroot}${prefix}/lib/libtommath.dylib
+    exec sed -e "s,^prefix=.*,prefix=${prefix}," -e "s,^Version:.*,Version: ${version}," \
+        -e "s,^includedir=.*,includedir=${prefix}/include/${name}," \
+        ${worksrcpath}/libtommath.pc.in > ${worksrcpath}/libtommath.pc
```
Comment (ryandesign) [inline math/libtommath/Portfile:57]: Use `reinplace` instead of `exec sed` please.

**Comment by reneeotten** [comment]
according to upstream it will install a pkgconfig file when building shared libs. Any reason why we don't just do that instead?

**Comment by chrstphrchvz** [comment]
Switching to Makefile.shared (as used by Homebrew) does seem ideal, but I don’t know how much more effort that will require. So for now I have adapted the steps in the makefiles for installing the pkgonfig file to the destroot phase of the portfile.

**Comment by ryandesign** [comment]
> according to upstream it will install a pkgconfig file when building shared libs. Any reason why we don't just do that instead?

The port does already install dynamic libraries.

**Comment by ryandesign** [comment]
> The port does already install dynamic libraries.

But does so using a patch we're adding to the makefile. If they have a makefile that builds dynamic libraries and installs pkg-config files for us without needing such patches, that would be great.

**Comment by kencu** [comment]
there’s also a nice-looking cmake build script that looks tempting

**Comment by chrstphrchvz** [comment]
> there’s also a nice-looking cmake build script that looks tempting

It has yet to be released, though.

**Pull request merged**